dev_tools/conf/apt-list-dev-tools.txt

```diff
@@ -1,2 +1,4 @@
 virtualenvwrapper
 pandoc
+docker-ce
```

Comment:
Nit: leave a comment here indicating these are currently Rigetti only.

Reply:
If I add a comment here it will be necessary to strip these out when doing the apt-get xargs command that is described in development.md

I'll add a comment in development.md